src/main/scala/ToQuery.scala

```diff
@@ -0,0 +1,11 @@
+package com.originate.scalypher
+
+import types._
```

Comment:
I like using this style of import too (where the full path isn't qualified) but there are other files that do the whole `com.originate.scalypher.foo.bar` import like `Action.scala`. It'd do us good to pick one style or the other and stick to it.